src/data/faq_de.json

```diff
@@ -204,7 +204,7 @@
                         "anchor": "ios_14_beta",
                         "active": true,
                         "textblock": [
-                            "Mit iOS 14 Beta 4 ist das Exposure Notification Framework inzwischen verfügbar. Dennoch ist diese Version <a href='https://developer.apple.com/support/beta-software/' target='_blank' rel='noopener noreferrer'>noch in Entwicklung</a> und ist nicht für den Betrieb auf produktiven Geräten vorgesehen. Die Corona-Warn-App wird derzeit auf iOS 14 Beta 4 noch nicht unterstützt. Siehe <a href='https://developer.apple.com/documentation/ios-ipados-release-notes/ios-ipados-14-beta-release-notes' target='_blank' rel='noopener noreferrer'>iOS & iPadOS 14 Beta Release Notes</a>."
+                            "Auch wenn mit iOS 14 Beta 4 das Exposure Notification Framework inzwischen verfügbar ist, unterstützt die Corona-Warn-App die iOS 14 Beta-Versionen nicht. Die Beta-Versionen sind <a href='https://developer.apple.com/support/beta-software/' target='_blank' rel='noopener noreferrer'>noch in Entwicklung</a> und ist nicht für den Betrieb auf produktiven Geräten vorgesehen. Siehe <a href='https://developer.apple.com/documentation/ios-ipados-release-notes/ios-ipados-14-beta-release-notes' target='_blank' rel='noopener noreferrer'>iOS & iPadOS 14 Beta Release Notes</a>."
```

Comment:
I'm so sorry but I just now saw that ther is a mistake in the new text. 
Here the right one:

>Auch wenn mit iOS 14 Beta 4 das Exposure Notification Framework inzwischen verfügbar ist, unterstützt die Corona-Warn-App die iOS 14 Beta-Versionen nicht. Die Beta-Versionen sind <a href='https://developer.apple.com/support/beta-software/' target='_blank' rel='noopener noreferrer'>noch in Entwicklung</a> und **sind** nicht für den Betrieb auf produktiven Geräten vorgesehen. Siehe <a href='https://developer.apple.com/documentation/ios-ipados-release-notes/ios-ipados-14-beta-release-notes' target='_blank' rel='noopener noreferrer'>iOS & iPadOS 14 Beta Release Notes</a>.